ansible/roles/test/files/helpers/bfd_responder.py

```diff
@@ -0,0 +1,188 @@
+import binascii
+import socket
+import struct
+import select
+import json
+import argparse
+import os.path
+from fcntl import ioctl
+from pprint import pprint
+import logging
+logging.getLogger("scapy.runtime").setLevel(logging.ERROR)
+import ptf.packet as scapy
+import scapy.all as scapy2
+scapy2.conf.use_pcap=True
+import scapy.arch.pcapdnet
+from  scapy.contrib.bfd import BFD
+def hexdump(data):
+    print " ".join("%02x" % ord(d) for d in data)
```

Comment:
Can you make this print compatible with python3?

Reply:
done